app/controllers/simulator_controller.rb

```diff
@@ -10,7 +10,7 @@ class SimulatorController < ApplicationController
   before_action :check_view_access, only: %i[show embed get_data]
   before_action :check_edit_access, only: %i[edit update update_image]
   skip_before_action :verify_authenticity_token, only: %i[get_data create]
-  after_action :allow_iframe, only: :embed
+  after_action :allow_iframe, only: %i[show embed]
```

Comment:
There is a reason why we have different iframe options for embed and show. I don't think we should allow iframe for all show requests. 
We have 2 options:
1. Allow iFrame Embed for show just for LTI sessions
2. Use embed route instead of show route for LTI sessions

I think 1st one is neater. Alternatively, we can allow iFrame for all LTI sessions.

Reply:
sure then i will do the 1st one

Reply:
done